src/ejp_uat.js

```diff
@@ -867,48 +868,59 @@ function ejpUatParseXmlTerms(url, xmlString) {
         // Init this in the terms list
         for ( var j = 0; j < termNode.childNodes.length; j++ ) {
             tagName = termNode.childNodes[j].tagName;
-            if ( tagName == 'prefLabel' ) {
+
+            if ( !tagName ) continue; // KF - skip if we can't find tagName
+            if (tagName.includes('prefLabel')) {  // KF
+            //if ( tagName == 'prefLabel' ) {
```

Comment:
Likewise, with good version control systems like Git, we can be bolder about getting rid of old code and not just commenting it out. With Git's really good tools for investigating history, we can put more of an emphasis on keeping code readable and trust Git to take care of the change-tracking bits.